sycl/doc/extensions/experimental/sycl_ext_codeplay_max_registers_per_work_group_query.md

```diff
@@ -0,0 +1,49 @@
+# sycl_ext_codeplay_max_registers_per_work_group_query
+
+## Notice
+
+This document describes an **experimental** API that applications can use to try
+out a new feature. Future versions of this API may change in ways that are
+incompatible with this experimental version.
+
+
+## Introduction
+
+This extension adds functionally a new device information descriptor. It provides the ability to query a device for the maximum number of registers available per work-group.
```

Comment:
Nit; I think this reads a little better.
```suggestion
This extension adds a new device information descriptor that provides the ability to query a device for the maximum number of registers available per work-group.
```

Reply:
Does sound much clearer. Appreciated!